chain/indexer-primitives/src/lib.rs

```diff
@@ -8,7 +8,6 @@ pub use near_primitives::{types, views};
 pub struct StreamerMessage {
     pub block: views::BlockView,
     pub shards: Vec<IndexerShard>,
```

Comment:
do you have to worry about backwards compatibility ? is this message used in communication between two different binaries ?

Reply:
A little bit above in this PR we have talked about it, let me know if that doesn't answer your question

https://github.com/near/nearcore/pull/6255#discussion_r801527633